db/migrate/20160622154200_create_users.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
Not really related to this file. But how did we get such an old migration?

Reply:
Another gift from that script repo...